src/Sulu/Bundle/ContentBundle/DependencyInjection/Configuration.php

```diff
@@ -52,6 +52,15 @@ public function getConfigTreeBuilder()
                         ->booleanNode('websocket')->defaultFalse()->end()
                     ->end()
                 ->end()
+                ->arrayNode('templates')
```

Comment:
@dantleech I have also just encountered that... But actually they are misplaced, they should belong to the `sulu_content`-configuration. Maybe we can move them later also to this place?